### PR TITLE
adds developer update september

### PR DESCRIPTION
**Pull request by SvenMuller95**

### CODE REVIEW

File: blog/2024-10-02-envio-community-update-sep-2024.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+<!--truncate-->
+
+
+Welcome to our September 2024 developer update!
+
+We’re back with another exciting update on the latest and greatest advancements from the Envio team. In this update, we’ll explore our recent release, V2.3.0, highlighting innovative features like [Wildcard Indexing](https://docs.envio.dev/docs/HyperIndex/wildcard-indexing), [IPFS](https://docs.envio.dev/docs/HyperIndex/ipfs) integration, and more. We’ll also recap key updates from September, including our time at [Token2049](https://www.asia.token2049.com/) and [EthCapeTown](https://www.ethcapetown.com/), upcoming events, partnerships, and other important developments.
+
+## 🚀 Version 2.3.0 is now available🚀
+
+We are excited to announce that the current release is **v.2.3.2**!
```
Comment (moose-code) [inline blog/2024-10-02-envio-community-update-sep-2024.md:18]: current release is actually 2.4.1 :)

Comment (moose-code) [reply blog/2024-10-02-envio-community-update-sep-2024.md:18]: But can leave that feature spotlight for next month

**Review by moose-code — APPROVED** [review]
Nice!!! Just to change the current latest version please

**Comment by Jordy-Baby** [comment]
Thanks @moose-code - updated 🙏

**Pull request merged**